### PR TITLE
chore(python,wasm): Replace lazy_static with once_cell

### PR DESCRIPTION
**Pull request by PsiACE**
Part of #1

### CODE REVIEW

**Comment by Xuanwo** [comment]
Seems deno did some breaking changes, I will take a look at https://github.com/risingwavelabs/arrow-udf/issues/38

File: Cargo.toml [hunk 1]
```diff
@@ -13,3 +13,6 @@ members = [
     "arrow-udf-python",
     "arrow-udf-wasm",
 ]
+
+[patch.crates-io]
+bit-set = { git = "https://github.com/contain-rs/bit-set.git" }
```
Comment (PsiACE) [inline Cargo.toml:18]: Let's temporarily patch it.

Comment (Xuanwo) [reply Cargo.toml:18]: Should be fixed in https://github.com/risingwavelabs/arrow-udf/pull/39

**Review by Xuanwo — APPROVED** [review]
Thanks!

**Pull request merged**